python/lsst/ts/gis/commander.py

```diff
@@ -31,39 +56,61 @@ def connected(self):
         Returns
         -------
         `bool`
+            Is the client connected?
         """
-        if self.client is None or not self.client.is_socket_open:
+        if self.client is None or not self.client.connected:
```

Comment:
I think lines 61 to 64 can be simplified with `return self.client is not None and self.client.connected`.